### PR TITLE
DT_DEPLOYMENT_METADATA env var added to AG PODs

### PR DESCRIPTION
**Pull request by aorcholski**
DT_DEPLOYMENT_METADATA environment variable added

### CODE REVIEW

File: controllers/activegate/statefulset.go [hunk 1]
```diff
@@ -195,10 +197,13 @@ func buildVolumeMounts(stsProperties *statefulSetProperties) []corev1.VolumeMoun
 }
 
 func buildEnvs(stsProperties *statefulSetProperties) []corev1.EnvVar {
```
Comment (mjgrzybek) [inline controllers/activegate/statefulset.go:199]: it's used onfly for `containers`, but not for `initContainers`. Open question - should we provide this env vars to `initContainers`? I think yes.

Comment (aorcholski) [reply controllers/activegate/statefulset.go:199]: There are no initContainers yet. Will be added by PR "Feature/k8s internal certificate import to the operator". So let's do it after the PR is merged.

**Review by DTMad — DISMISSED** [review]

File: controllers/activegate/statefulset.go [hunk 2]
```diff
@@ -38,6 +39,7 @@ const (
 	DTNetworkZone     = "DT_NETWORK_ZONE"
 	DTGroup           = "DT_GROUP"
 	DTInternalProxy   = "DT_INTERNAL_PROXY"
+	DTDeployMetadata  = "DT_DEPLOY_METADATA"
```
Comment (baichinger) [inline controllers/activegate/statefulset.go:42]: The title of the PR says _DT_DEPLOYMENT_METADATA_, for code modules it is _DT_DEPLOYMENT_METADATA_ as well. Does ActiveGate really use a different one?

**Review by aorcholski — COMMENTED** [review]
Yes, you are right. Fix committed.

**Review by DTMad — APPROVED** [review]

**Pull request merged**